pkg/registry/replacesgraphloader.go

```diff
@@ -15,11 +15,21 @@ func (r *ReplacesGraphLoader) CanAdd(bundle *Bundle, graph *Package) (bool, erro
 		return false, fmt.Errorf("Invalid content, unable to parse bundle")
 	}
 
+	// adding the first bundle in the graph
+	if replaces == "" {
+		return true, nil
+	}
+	csvName := bundle.Name
```

Comment:
nit: `csvName` is only used once, so we should probably omit the local variable in favor of `bundle.Name`.